src/ontology/mondo-edit.obo

```diff
@@ -459200,18 +459200,16 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/596/albright
 
 [Term]
 id: MONDO:0022414
-name: allain-babin-demarquez syndrome
-comment: Reason of obsoletion: out of scope - MONDO:excludeObsoleteSource. Term to consider: -
-subset: obsoletion_candidate
+name: obsolete allain-babin-demarquez syndrome
 subset: orphanet_rare {source="Orphanet:1526"}
```

Comment:
@sabrinatoro should this subset tag also be removed?

Reply:
@nicolevasilevsky I think this should be removed based on the Slack conversation on Wednesday, March 13.